chapters/101.rst

```diff
@@ -4,6 +4,8 @@ Jina 101: Basic components
 
 
 In Jina 101, you will learn about the basic components of Jina. Before you start, read `the basics of Neural Search`_ to get some context.
+You can install Jina with PyPI and run `jina hello` demos to see examples on fashion image search, chatbot and multi modality search. Upgrade to  the latest version by using `pip install -U jina`
+
```

Comment:
Let's change this to:

In Jina 101, you will learn about the basic components of Jina. Before you start, read `the basics of Neural Search`_ to get some context and run Jina's [Hello World demos](https://github.com/jina-ai/jina/blob/master/.github/pages/hello-world.md) to see Jina in action.

**Note:** I've mixed markdown and rst in the snippet above. Please convert to pure rst format when you modify the file